orchagent/qosorch.cpp

```diff
@@ -813,28 +819,35 @@ task_process_status QosOrch::handleSchedulerTable(Consumer& consumer)
                 // TODO: The meaning is to be able to adjus priority of the given scheduler group.
                 // However currently SAI model does not provide such ability.
             }
+            else if (fvField(*i) == scheduler_meter_type_field_name)
+            {
+                sai_meter_type_t meter_value = scheduler_meter_map.at(fvValue(*i));
+                attr.id = SAI_SCHEDULER_ATTR_METER_TYPE;
+                attr.value.s32 = meter_value;
+                sai_attr_list.push_back(attr);
+            }
             else if (fvField(*i) == scheduler_min_bandwidth_rate_field_name)
             {
                 attr.id = SAI_SCHEDULER_ATTR_MIN_BANDWIDTH_RATE;
-                attr.value.u64 = stoul(fvValue(*i));
+                attr.value.u64 = stoull(fvValue(*i));
```

Comment:
what is the reason to use unsigned long long?

Reply:
For shaping rates above ~34G, we go over 32 bits to store the value.  This is reflected in usage of u64 in the SAI type.  I'm explicitly doing the same for the string conversion function to return u64 as well.